src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,43 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-size + $gap-size) * 7 - $gap-size + $padding-size * 2;
+  padding: $padding-size;
```

Comment:
Add bigger padding on the top, because when I hover over the element, it goes out of screen.